qiskit/visualization/pulse_v2/style/default.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
I think we should be weary about introducing a new data format. Is this defined anywhere else? Would it be possible to define just as a python dict for now that can be dumped to JSON?

Reply:
I'm going to replace the json to python dict. The json file is not necessary.